ui/components/multichain/pages/send/index.scss

```diff
@@ -2,6 +2,7 @@
 
 .multichain-send-page {
   width: 100%;
+  max-width: 480px;
```

Comment:
This aligns with our container component and the [max width set](http://localhost:6006/?path=/docs/components-componentlibrary-container--docs#maxwidth). The reason I didn't use the Container component is that it would require a fair amount of refactoring.

<img width="757" alt="Screenshot 2024-06-25 at 9:41:19 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/db424a79-50e7-4f32-9f56-1c6446524e0a">